src/web/routes/application/common/confirmation-code.js

```diff
@@ -0,0 +1,16 @@
+// reset confirmation code and reset state to in progress so that the user is not taken straight back to check details.
```

Comment:
Shouldn't this comment be against the handleConfirmationCodeReset method?

Reply:
Fixed.

Reply:
it hasn't moved yet...

Reply:
Not sure what happened there, fixed now.